src/color/brand.json

```diff
@@ -478,6 +478,52 @@
           "wcag": "n/a",
           "deprecated": false
         }
+      },
```

Comment:
The other token declarations have some keys that aren't included here:

comment
neutral
wcag
deprecated

I'm not sure why the objects aren't consistent or if that is a problem or not?

Reply:
Dale went through them and told me to ignore them.